pkg/config/dynamicconfig.go

```diff
@@ -284,7 +292,13 @@ func (cd *ConfigData) load(cm v1.ConfigMap) (reconcilePolicyReport, updateWebhoo
 
 	webhooks, ok := cm.Data["webhooks"]
 	if !ok {
-		logger.V(4).Info("configuration: No webhook configurations defined in ConfigMap")
+		if len(cd.webhooks) > 0 {
+			cd.webhooks = nil
+			updateWebhook = true
```

Comment:
>updateWebhook = true

Is it to remove `namespaceSelector` configs from the webhook configurations? If so, suggest updating the log message to:
> configuration: setting namespaceSelector to empty in the webhook configurations

Reply:
Got it, will update as suggested.